strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntityHook.java

```diff
@@ -68,10 +74,22 @@ else if (artifactCoordinates != null && !artifactEntryPath.equals(artifactCoordi
                             String.format("Failed to persist document [%s]. Paths [%s] and [%s] dont match.",
                                           doc.getSchemaClass(), artifactEntryPath, artifactCoordinatesPath));
                 }
+
+                validateArtifactEntryCreatedProperty(doc);
             }
         }
 
         return result;
     }
 
+    private void validateArtifactEntryCreatedProperty(ODocument doc) {
+        Date created = doc.field("created");
+        if (created == null)
+        {
+            throw new OValidationException(
```

Comment:
Cosmetic/Codestyle: The parameters should be aligned to the method's bracket, like this:
```
            throw new OValidationException(String.format("Failed to persist document [%s]. 'created' can't be null.",
                                           doc.getSchemaClass()));
```